staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -97,7 +97,7 @@ func WithLogging(handler http.Handler, pred StacktracePred) http.Handler {
 		req = req.WithContext(context.WithValue(ctx, respLoggerContextKey, rl))
 
 		if klog.V(3).Enabled() {
-			defer func() { klog.InfoS("HTTP", rl.LogArgs()...) }()
+			defer func() { klog.InfoSDepth(1, "HTTP", rl.LogArgs()...) }()
```

Comment:
Without InfoSDepth we needed to have the log call in the same function. Using InfoSDepth allows us to move it back to to dedicated function. 
PTAL https://github.com/kubernetes/kubernetes/pull/91833/files

Reply:
@serathius I moved it into a dedicated function using a similar implementation used before, can you check now? 

Let me know if you have any further suggestions ( I am fairly new to this :P )